.github/CODEOWNERS

```diff
@@ -27,17 +26,18 @@
 # Integrations
 /integrations/ @benvanik @stellaraccident
 /integrations/tensorflow/ @stellaraccident
-/integrations/tensorflow/build_tools/ @GMNGeoffrey @stellaraccident
 /integrations/tensorflow/test/**/iree_tfl_tests/ @rsuderman
 
 # Experimental
 # It's experimental, but we still don't want any old directory added here.
 /experimental/ @benvanik @GMNGeoffrey @stellaraccident
-/experimental/remoting/ @stellaraccident
+/experimental/cpu_ukernel/ @bjacob
+/experimental/cuda2/ @antiagainst
+/experimental/dispatch_profiler/ @manishucsd
+/experimental/rocm/ @benvanik
 /experimental/web/ @ScottTodd
```

Comment:
So I'll add `/experimental/webgpu/` here in https://github.com/openxla/iree/pull/13952, rather than in this PR?

Reply:
Yes please :-)

Reply:
You'll still get tagged for `/.github/ @GMNGeoffrey @ScottTodd` ;P

Reply:
Well I'd get tagged for the addition of the directory anyway, assuming it uses the checked-in CODEOWNERS and doesn't override it with the one from the PR (which would be absolutely bananas)

Reply:
But getting added once for directory creation is fine: it's getting added for every PR that is getting annoying